src/v2/Apps/Artist/Components/ArtistTopWorksRail/ArtistTopWorksRail.tsx

```diff
@@ -0,0 +1,150 @@
+import { Box, Flex, Sans } from "@artsy/palette"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { ArtistTopWorksRail_topRailArtworks } from "v2/__generated__/ArtistTopWorksRail_topRailArtworks.graphql"
+import { Carousel } from "v2/Components/Carousel"
+import FillwidthItem from "v2/Components/Artwork/FillwidthItem"
+import { Media } from "v2/Utils/Responsive"
+import { StyledLink } from "v2/Apps/Artist/Components/StyledLink"
+
+interface ArtistTopWorksRailProps {
+  topRailArtworks: ArtistTopWorksRail_topRailArtworks
```

Comment:
This prop name feels a bit off, but I don't have a great suggestion. The component name `ArtistTopWorksRail` feels good, should this also be `topArtworksRail`, or just `artworks` or something like that?

Reply:
Looking below, what's being passed in [here](https://github.com/artsy/force/pull/5647/files#diff-15191c82a544ca984d0ce06cf50be900R188) is `artist`, so conventionally the prop is also called `artist` since its spread on the `Artist` fragment.

Reply:
ah yea, good call.